FaunaDB.Client.Test/SerializationTest.cs

```diff
@@ -1090,5 +1091,26 @@ public void TestDocuments()
         {
             AssertJsonEqual(Documents(Collection("foo")), "{\"documents\":{\"collection\":\"foo\"}}");
         }
+
+        [Test]
+        public void TestEncodeRawExpressions()
+        {
+            var indexCfg = new Dictionary<string, Expr>()
+            {
+                { "name", "index_name" },
+                { "source", Collection("class_name") }
+            };
+
+            AssertJsonEqual(
+                Encode(indexCfg),
+                "{\"object\":{\"name\":\"index_name\",\"source\":{\"collection\":\"class_name\"}}}"
+           );
+
+            AssertJsonEqual(
+                CreateIndex((Expr)indexCfg),
```

Comment:
I don't remember, but does the cast `(Expr)` really necessary?

Reply:
you're write... we don't need that cast